libs/@guardian/libs/package.json

```diff
@@ -69,13 +68,23 @@
 			]
 		},
 		"e2e": {
-			"command": "cypress run --headless",
+			"command": "pnpm exec playwright install chromium && playwright test",
+			"dependencies": [
+				"../../../apps/github-pages:start"
+			],
 			"files": [
+				"**",
 				"!(dist)/**",
-				"!(.eslintcache)",
-				"!(jest*)"
+				"!.eslintcache",
+				"!jest*",
+				"!(playwright-report)/**",
+				"!(test-results)/**",
+				"!*.md"
 			],
-			"output": [],
+			"output": []
+		},
+		"e2e:ui": {
```

Comment:
Does this need ` files` and `output` defined like the `e2e` task?

Reply:
I dont think so because they are for caching. I don't think we want to cache this one as it is a debug mode for the tests - https://github.com/google/wireit/blob/main/README.md#caching

Reply:
Ah, yes, that makes sense! 😄